src/components/Sidebar.tsx

```diff
@@ -24,6 +24,24 @@ const searchStringInArray = (str: string, allChains: Chains) => {
       (allChains[a].testnet && allChains[b].testnet) ||
       (!allChains[a].testnet && !allChains[b].testnet)
     ) {
+      if (a == "polkadot") {
```

Comment:
Nice =)
Could have been better, but works, can leave it as is

Reply:
We'll make a full redesign of the portal when Omni will go live and we'll hire more developers.